packages/microsite/README.md

```diff
@@ -56,3 +56,31 @@ project/
 - [Lukas Bombach](https://twitter.com/luke_schmuke), [The case of partial hydration (with Next and Preact)](https://medium.com/@luke_schmuke/how-we-achieved-the-best-web-performance-with-partial-hydration-20fab9c808d5)
 - [Jason Miller](https://twitter.com/_developit) and [Addy Osmani](https://twitter.com/addyosmani), [Rendering on the Web](https://developers.google.com/web/updates/2019/02/rendering-on-the-web)
 - [Poimandres](https://github.com/pmndrs), [`valtio`](https://github.com/pmndrs/valtio) for inspiring `microsite/global`
+
+## Contributing
+
+This is a monorepo, set up with [Lerna](https://github.com/lerna/lerna), you should start by familiarizing yourself with its commands, as it's a little bit different from standard Node/NPM workflows. The most important thing to remember is that commands should generally be run from the monorepo root directory.
+
+### Initial dev setup
+
+1. Install [NVM](https://github.com/nvm-sh/nvm/) if you haven't already, to ensure you're using the Node/NPM version specified by the project.
+
+2. Run NVM: `nvm use`.
+
+3. Install dependencies for the monorepo: `npm install`.
+
+4. Install and link dependencies in sub-packages: `npx lerna bootstrap`. <!-- Note for review: I actually have no idea which command is preferred here, each one I've tried has still produced changes to most or all `package-lock.json` files in the project. -->
```

Comment:
Hmm, I'm not positive either. I think `npx lerna bootstrap --hoist`?

Reply:
I tried that and it had an even bigger impact on the package locks. I'll go with that if it's what you prefer, and now it'll be documented so we're both doing the same flow!

Reply:
I think for the normal dev setup this is probably the preferred solution, in that it minimizes the hit to the file system. But I will note that it doesn’t play nice with `npm install` in the root package, because it wipes out most of the root lock file. This isn’t a problem I’ve had using Yarn with workspaces, because both tools know about each other’s hoist behavior. However another call to bootstrap does work. It’s just a somewhat slow operation.

Reply:
Hmm, weird. I would be open to using `yarn` if it would solve this? Volta manages your package managers (`npm`, `yarn`, etc) 

Also I have my eye on [`turborepo`](https://turborepo.com/) to solve a lot of `lerna` headaches... Not the biggest `lerna` fan but it seems to be the least terrible option for now.

Reply:
I'm hesitant to recommend Yarn due to issues I mentioned [here](https://github.com/natemoo-re/microsite/pull/100#discussion_r563182855), even though I'm pretty familiar with it, and have even somewhat validated using it in a monorepo _without_ any other tools. Most of my own projects are currently pinned to Yarn 1 to avoid those PnP issues, and I don't think there's a future where I'll want to migrate them to Yarn 2 because of those issues, so for my use Yarn is only viable as long as v1 is actively being maintained. The other alternative I mentioned in that comment, PNPM, also supports workspaces, so I would imagine it's also well positioned to play well in a monorepo.

I'm not familiar with Turborepo, so I can't speak to that. Another one I've wanted to look into is [Rush](https://rushjs.io/), which is developed by Microsoft (they definitely do monorepos!), and supports NPM, PNPM, and Yarn.